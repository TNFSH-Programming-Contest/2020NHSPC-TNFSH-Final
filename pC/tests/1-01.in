5 10 4
1 4 88761
1 2 790600
2 5 533908
5 3 555760
2 3 95565
5 1 245232
4 2 493609
3 4 81770
5 4 905135
3 1 106875
