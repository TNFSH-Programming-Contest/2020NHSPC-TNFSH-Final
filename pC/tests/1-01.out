88761 354670 81770 0 667986 
